fauntlet/src/main.rs

```diff
@@ -0,0 +1,178 @@
+
+    // Locations in normalized variation space
+    let var_locations = [-1.0, -0.32, 0.0, 0.42, 1.0].map(|c| F2Dot14::from_f32(c));
+
+    use clap::Parser as _;
+    let args = Args::parse_from(wild::args());
+
+    match args.command {
+        Command::CompareGlyphs {
+            print_paths,
+            exit_on_fail,
+            files,
+        } => {
+            use std::sync::atomic::{AtomicBool, Ordering};
+            let ok = AtomicBool::new(true);
+            files.par_iter().for_each(|font_path| {
+                if print_paths {
+                    writeln!(std::io::stdout(), "[{font_path:?}]").unwrap();
+                }
+                if let Some(mut font_data) = fauntlet::Font::new(&font_path) {
+                    for font_ix in 0..font_data.count() {
+                        for ppem in &ppem_sizes {
+                            let axis_count = font_data.axis_count(font_ix) as usize;
+                            if axis_count != 0 {
+                                let mut coords = vec![];
+                                for coord in &var_locations {
+                                    coords.clear();
+                                    coords.extend(std::iter::repeat(*coord).take(axis_count));
+                                    let options =
+                                        fauntlet::InstanceOptions::new(font_ix, *ppem, &coords);
+                                    if let Some(instances) = font_data.instantiate(&options) {
+                                        if !compare_glyphs(
+                                            &font_path,
+                                            &options,
+                                            instances,
+                                            exit_on_fail,
+                                        ) {
+                                            ok.store(false, Ordering::Release);
+                                        }
+                                    }
+                                }
+                            } else {
+                                let options = InstanceOptions::new(font_ix, *ppem, &[]);
+                                if let Some(instances) = font_data.instantiate(&options) {
+                                    if !compare_glyphs(
+                                        &font_path,
+                                        &options,
+                                        instances,
+                                        exit_on_fail,
+                                    ) {
+                                        ok.store(false, Ordering::Release);
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            });
+            if !ok.load(Ordering::Acquire) {
+                std::process::exit(1);
+            }
+        }
+    }
+}
+
+fn compare_glyphs(
```

Comment:
Optional: Could this be in a separate file to keep the "test driver" logic separate from the comparison logic?

Reply:
Looks done, thx.